src/main/scala/org/globalforestwatch/layers/TreeCoverLossDrivers.scala

```diff
@@ -5,7 +5,9 @@ import org.globalforestwatch.grids.GridTile
 case class TreeCoverLossDrivers(gridTile: GridTile)
     extends StringLayer
     with OptionalILayer {
-  val uri: String = s"$basePath/tsc_tree_cover_loss_drivers/v2020/raster/epsg-4326/${gridTile.gridSize}/${gridTile.rowCount}/type/gdal-geotiff/${gridTile.tileId}.tif"
+  val uri: String =
+//    s"$basePath/tsc_tree_cover_loss_drivers/v2020/raster/epsg-4326/${gridTile.gridSize}/${gridTile.rowCount}/type/gdal-geotiff/${gridTile.tileId}.tif"
+    s"s3://gfw-files/flux_1_2_1/drivers/${gridTile.tileId}.tif"
```

Comment:
Is this the new drivers data? We should probably put this in the data lake

Reply:
It is the new drivers data and it should be added to the datalake but Jimmy hasn't confirmed with TSC that it's final yet and therefore Maanas hasn't ingested it yet. I didn't want to wait for them to do all that to get my flux model statistics, so this was my stopgap measure. Just like the updated flux model outputs, this URI will need to be updated with the 2020 version when Maanas has put it through the raster pipeline.